src/sources/multisrc/wpmangastream/WPMangaStreamScraper.js

```diff
@@ -91,12 +91,12 @@ class WPMangaStreamScraper {
     novel.genre = loadedCheerio('.genxed').text().trim().replace(/\s/g, ',');
 
     novel.summary = loadedCheerio('.entry-content')
-      .find('h3')
+      .find('h3 , p.a')
```

Comment:
I checked on the source sites, wouldn't `div[itemprop="description"]` be a better selector?

```
novel.summary = loadedCheerio('div[itemprop="description"]').text()
```

Reply:
I basically want to make the summary look cleaner by removing those within the box
![Screenshot from 2023-02-22 12-14-33](https://user-images.githubusercontent.com/26593485/220521542-6c02ddc4-e34d-4e6b-b2f1-ee610d478e50.png)

so with this
``novel.summary = loadedCheerio('div[itemprop="description"]').text()``
it looks like this
![Screenshot from 2023-02-22 12-26-16](https://user-images.githubusercontent.com/26593485/220521889-a59cd683-3004-4dc7-9657-18550f97ecaa.png)

so I try to rectify it (somewhat, I wish I could do better) with this,
`` novel.summary = loadedCheerio('.entry-content')
      .find('h3 , p.a')
      .remove()
      .end()
      .prop('innerHTML')
      .replace(/(<.*?>)/g, ' ')
      .replace(/(&.*;)/g, '\n');``
      
so it looks something like this,
![Screenshot from 2023-02-22 12-30-15](https://user-images.githubusercontent.com/26593485/220522304-0c82dc82-dd87-446b-bd8a-e9f7990a5781.png)